update: Fix information on Android distributions page (#2844)
privacyguides/privacyguides.org: docs/os/android-overview.md

On Android distributions with [Sandboxed Google Play](https://grapheneos.org/usage#sandboxed-google-play), go to :gear: **Settings** → **Apps** → **Sandboxed Google Play** → **Google Settings** → **Ads**, and select *Delete advertising ID*.

> On Android distributions with privileged Google Play Services (such as stock OS's), the setting may be in one of several locations. Check

Reviewer: ```suggestion
On Android distributions with privileged Google Play Services (which includes the stock installation on most devices), the setting may be in one of several locations. Check
```
